js/logging.js

```diff
@@ -91,10 +91,10 @@ loggingModule.factory(
     "applicationLoggingService",
     ["$log","$window", "LOGGING_CONFIG", function($log, $window, LOGGING_CONFIG){
         return({
-            error: function(message){
+            error: function(message, desc){
                 if (LOGGING_CONFIG.LOGGING_TYPE !== 'none') {
                   // preserve default behaviour
-                  $log.error.apply($log, arguments);
```

Comment:
https://docs.nodejitsu.com/articles/javascript-conventions/what-is-the-arguments-object - magic